.github/workflows/ci.yaml

```diff
@@ -51,3 +51,4 @@ jobs:
     uses: ./.github/workflows/e2e.yaml
     with:
       version: ${{ needs.variables.outputs.version }}
+      distribution: ubuntu20.04
```

Comment:
Given that we're working on #602 I think we should just set the distribution in the test.

Reply:
Done